varintf/varintf_test.go

```diff
@@ -95,6 +96,34 @@ func TestFactomSpecExamples(t *testing.T) {
 	}
 }
 
+// TestVarInt_Panic ensures the varint decoding will never panic from invalid data
+func TestVarInt_Panic(t *testing.T) {
```

Comment:
I updated varintf to avoid panics. Haven't tested it against this yet.

Reply:
Test passes now.